airflow-core/src/airflow/dag_processing/processor.py

```diff
@@ -94,8 +96,35 @@ def _parse_file_entrypoint():
         comms_decoder.send_request(log, result)
 
 
+def _pre_import_airflow_modules(file_path: str, log: FilteringBoundLogger) -> None:
+    """
+    Pre-import Airflow modules found in the given file.
+
+    This prevents modules from being re-imported in each processing process,
+    saving CPU time and memory.
+
+    Args:
+        file_path: Path to the file to scan for imports
+        log: Logger instance to use for warnings
+
+    parsing_pre_import_modules:
+         default value is True
+    """
+    if not conf.getboolean("scheduler", "parsing_pre_import_modules", fallback=True):
```

Comment:
This should be a dag processor config and not scheduler

Reply:
I think what @ephraimbuddy means is something like `if not conf.getboolean("dag_processor", "parsing_pre_import_modules", fallback=True):`